content/manuals/engine/logging/drivers/awslogs.md

```diff
@@ -127,6 +127,30 @@ $ docker run \
 > Your AWS IAM policy must include the `logs:CreateLogGroup` permission before
 > you attempt to use `awslogs-create-group`.
 
+### awslogs-create-stream
+
+By default, the log driver will attempt to create the AWS CloudWatch Logs stream to be used
```

Comment:
non-blocking; we try to avoid future tense in our docs.

Reply:
😅 should be addressed in https://github.com/docker/docs/pull/20928/commits/4c60d767335ccf6d1168c34ad4df6a72b5e3c441